style-guide/assets/wds-icons-tag.svg

```diff
@@ -0,0 +1,22 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="15" height="15" viewBox="0 0 15 15">
+  <defs>
+    <path id="a" d="M7.368 13.936L2.064 8.632C1.19 7.758.5 6.096.5 4.86V3C.5 1.616 1.618.5 3 .5h1.86c1.241 0 2.894.686 3.772 1.564l5.304 5.304a2.51 2.51 0 0 1-.001 3.544l-3.023 3.023a2.498 2.498 0 0 1-3.544 0zm.707-.707a1.5 1.5 0 0 0 2.13-.001l3.023-3.023a1.51 1.51 0 0 0 0-2.13L7.926 2.771C7.234 2.081 5.835 1.5 4.86 1.5H3c-.83 0-1.5.668-1.5 1.5v1.86c0 .97.585 2.378 1.271 3.065l5.304 5.304z"/>
+    <path id="c" d="M4.907 6.71a1.802 1.802 0 1 1 0-3.605 1.802 1.802 0 0 1 0 3.604zm0-1a.802.802 0 1 0 0-1.605.802.802 0 0 0 0 1.604z"/>
+  </defs>
+  <g fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1 15 0)">
+    <mask id="b" fill="#fff">
+      <use xlink:href="#a"/>
+    </mask>
+    <use fill="#CCC" fill-rule="nonzero" xlink:href="#a"/>
+    <g fill="#999" mask="url(#b)">
+      <path d="M0 0h15v15H0z"/>
+    </g>
+    <mask id="d" fill="#fff">
+      <use xlink:href="#c"/>
+    </mask>
+    <use fill="#CCC" fill-rule="nonzero" xlink:href="#c"/>
+    <g fill="#999" mask="url(#d)">
+      <path d="M0 0h15v15H0z"/>
```

Comment:
This SVG seems pretty complicated in the code and simple visually. Maybe there is a way to simplify the code? @kvas-damian any suggestions?